addons/l10n_id/data/account.account.tag.csv

```diff
@@ -0,0 +1,7 @@
+id,name,applicability
```

Comment:
where are the tags used?

Reply:
in tax templates

Reply:
Ok, but they are not used in any report... (do they correspond to a vat return report?)